neoload/commands/project.py

```diff
@@ -29,8 +39,8 @@ def cli(command, name_or_id, path):
 #TODO: spider through all YAML (as-code files)
 #TODO: fix validate to recurse through all includes; create unique file list map (avoid recursive references)
 
-def upload(path, settings_id):
-    neoLoad_project.upload_project(path, get_endpoint(settings_id))
+def upload(path, settings_id, save):
+    neoLoad_project.upload_project(path, get_endpoint(settings_id), save=save)
```

Comment:
Fair, but I like to match parameter usage with how the target method signature specifies named parameters, rather than assume the positional on named parameters.

Reply:
Python manages both.

Reply:
Removed the named parameter syntax.